packages/nekovm/build.sh

```diff
@@ -0,0 +1,66 @@
+TERMUX_PKG_HOMEPAGE="https://nekovm.org/"
+TERMUX_PKG_DESCRIPTION="The Neko Virtual Machine"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2.3.0
+TERMUX_PKG_SRCURL=https://github.com/HaxeFoundation/neko/archive/refs/tags/v${TERMUX_PKG_VERSION//[.]/-}.tar.gz
+TERMUX_PKG_SHA256=850e7e317bdaf24ed652efeff89c1cb21380ca19f20e68a296c84f6bad4ee995
+TERMUX_PKG_DEPENDS="libgc,openssl,zlib,apache2,libsqlite,mbedtls"
+TERMUX_PKG_SRCDIR="../src"
+
+
+termux_step_configure_cmake() {
```

Comment:
???

You shouldn't override this step.

Read https://github.com/termux/termux-packages/wiki/Building-packages and also you have lots of sample packages in https://github.com/termux/termux-packages/tree/master/packages (including cmake-based).